templates/common/_base/files/mtu-migration-dispatcher.yaml

```diff
@@ -0,0 +1,213 @@
+{{ if .Network }}{{ if .Network.MTUMigration }}{{ if .Network.MTUMigration.Machine }}{{ if .Network.MTUMigration.Machine.To -}}
+mode: 0755
+path: "/etc/NetworkManager/dispatcher.d/pre-up.d/99-mtu-migration.sh"
+contents:
+  inline: |
+    #!/bin/bash
+
+    set -ex
+
+    MTU_MIGRATION_DIR="/etc/cno/mtu-migration"
+    MTU_MIGRATION_CONFIG="${MTU_MIGRATION_DIR}/config"
+    MTU_MIGRATION_SAVE_DIR="${MTU_MIGRATION_DIR}/mtu"
+
+    if [ "$2" != "pre-up" ]; then
+        set +x
```

Comment:
why do you need to do this?

Reply:
oops, since we're exiting, there's indeed no need.